src/locales/ko/mystery-encounters/bug-type-superfan-dialogue.json

```
@@ -1,38 +1,38 @@
-      "selected_2_to_3": "Hey, you've got {{numBugTypes}} Bug Types!\nNot bad.$Here, this might help you on your journey to catch more!",
-      "selected_4_to_5": "What? You have {{numBugTypes}} Bug Types?\nNice!$You're not quite at my level, but I can see shades of myself in you!\n$Take this, my young apprentice!",
-      "selected_6": "Whoa! {{numBugTypes}} Bug Types!\n$You must love Bug Types almost as much as I do!$Here, take this as a token of our camaraderie!"
+      "label": "벌레타입 포켓몬을 보여준다",
+      "tooltip": "(+) 아이템 획득",
+      "disabled_tooltip": "벌레타입 포켓몬이 1마리 이상 있어야 선택할 수 있다.",
+      "selected": "트레이너에게 가지고 있는 모든 벌레 포켓몬을 보여주었다...",
+      "selected_0_to_1": "응? 너 {{numBugTypes}}마리 갖고 있구나...$너같은 일반인한테 떠들다었다니, 산소가 아깝네...",
+      "selected_2_to_3": "어라, 너 {{numBugTypes}}마리 갖고 있구나!\n나쁘지 않아.$내가 더 많이 잡는데 도움이 될만 한 걸 줄게!",
+      "selected_4_to_5": "뭐야? 너 벌레타입을 {{numBugTypes}}마리나 갖고 있네?\좋아!$아직 수준엔 못 미치지만, 너한테 내 옛날 모습이 보여!\n$이걸 받거라, 내 제자여…!",
+      "selected_6": "우와! {{numBugTypes}}마리 모두 벌레타입!\n$너도 나만큼 벌레 타입을 사랑하는거지?$이거 가져가! 동지의 증표로 나눠줄게!"
     },
     "3": {
-      "label": "Gift a Bug Item",
-      "tooltip": "(-) Give the trainer a {{requiredBugItems}}\n(+) Receive a Gift Item",
-      "disabled_tooltip": "You need to have a {{requiredBugItems}} to select this.",
-      "select_prompt": "Select an item to give.",
-      "invalid_selection": "Pokémon doesn't have that kind of item.",
-      "selected": "You hand the trainer a {{selectedItem}}.",
-      "selected_dialogue": "Whoa! A {{selectedItem}}, for me?\nYou're not so bad, kid!$As a token of my appreciation,\nI want you to have this special gift!$It's been passed all through my family, and now I want you to have it!"
+      "label": "벌레 아이템을 선물한다",
+      "tooltip": "(-) 트레이너에게 {{requiredBugItems}}[[를]] 건네줌\n(+) 아이템 획득",
+      "disabled_tooltip": "{{requiredBugItems}}[[를]] 가지고 있어야 한다.",
+      "select_prompt": "건네줄 아이템을 선택하세요.",
+      "invalid_selection": "포켓몬이 해당하는 아이템을 갖고 있지 않다.",
+      "selected": "트레이너에게 {{selectedItem}}[[를]] 건넸다.",
+      "selected_dialogue": "우와! {{selectedItem}}, 나한테 주는거야?\n너 꽤 괜찮은 꼬마구나!$고마우니깐,\n나도 너한테 특별한 선물을 줄게!$가보로 전해져 내려온 귀한 물건이지만, 너한테 주고 싶어!"
     }
   },
-  "battle_won": "Your knowledge and skill were perfect at exploiting our weaknesses!$In exchange for the valuable lesson,\nallow me to teach one of your Pokémon a Bug Type Move!",
-  "teach_move_prompt": "Select a move to teach a Pokémon.",
-  "confirm_no_teach": "You sure you don't want to learn one of these great moves?",
-  "outro": "I see great Bug Pokémon in your future!\nMay our paths cross again!$Bug out!"
+  "battle_won": "너의 지식과 기술, 약점을 완벽하게 파악했잖아.$귀중한 교훈을 가르쳐줬으니,\n나도 벌레타입 기술을 하나 가르쳐 줄게!",
+  "teach_move_prompt": "기술을 가르칠 포켓몬을 선택하세요.",
+  "confirm_no_teach": "너, 정말로 이 대단한 기술을 하나도 배우지 않는거야?",
+  "outro": "I see great Bug Pokémon in your future!\nMay our paths cross again!$Bug out! --이거 트레이어 대사인지 아웃트로인지모르겠어요."
```

보통 이벤트는 outro 가 보통 3인칭 시점인데요.
이 이벤트는 상대가 1인칭시점으로 대사가 한번 더 있고 끝내는건지? 잘모르겠습니다

---

3인칭으로 나레이션으로 쓰는게 맞는것같습니다. ( 선택지 대사에 따라서는 이걸 1인칭으로 쓰면 너무 갑자기 태도가 바뀌는 느낌을 줄수도 있고, 인게임에서는 이 메시지는 speaker 가 안붙는 일반 메세지니까요. )
```suggestion
  "outro": "나중에 멋진 벌레 포켓몬을 만나길 바라며!\n다시 만날 때까지!$사사삭!"
```